src/stories/pages/404.stories.mdx

```diff
@@ -0,0 +1,17 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+import { linkTo } from '@storybook/addon-links';
+
+import { FourOhFour } from './404.stories';
+
+<Meta title="Pages/404/Intro" component={FourOhFour} />
+
+# 404
+
+A page view also known as an “error page” or “Page Not Found” page. This page
+indicates that the user reached the domain they requested, but the URL path
+provided no information.
```

Comment:
@dvdherron  Not a blocker for this pr, but I'm open to any ideas you have on component documentation. This is just something I copy/pasta'd from the web and we can include those changes in the [404 card](https://trello.com/c/f0jSvig0/93-metecho-404) in another branch,